### PR TITLE
feat: Enable Zeus

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
Also for context, since not everyone may know what goes on at Sentry, Zeus is a tool we're using internally to monitor build results:

http://cra.mr/2018/01/09/introducing-zeus

File: .travis.yml [hunk 1]
```diff
@@ -51,3 +51,23 @@ after_script:
   - if [ $(phpenv version-name) = "7.0" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover test/clover.xml --revision=$TRAVIS_COMMIT; fi
   - if [ $(phpenv version-name) = "7.1" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover test/clover.xml --revision=$TRAVIS_COMMIT; fi
   - if [ $(phpenv version-name) = "7.2" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover test/clover.xml --revision=$TRAVIS_COMMIT; fi
+
+after_success:
```
Comment (Jean85) [inline .travis.yml]: Since it's done anyway, couldn't you add it to the `after_script` section?

Comment (dcramer) [reply .travis.yml]: yeah can do that

Comment (dcramer) [reply .travis.yml]: @Jean85 it only happens on success, right?

Comment (dcramer) [reply .travis.yml]: i read docs it seems it happens no matter what

**Comment by dcramer** [comment]
Something odd is going on and I'm not seeing the webhooks update as expected in Zeus. Will debug before merging.

**Comment by Jean85** [comment]
Have you tested the webhook from the repo settings? It should have a `Test` button that should fire a webhook on demand.

**Comment by dcramer** [comment]
There's some incompatibility in Zeus that I need to resolve before this makes sense to merge.

For my own record keeping:

```
DETAIL:  Key (job_id, hash)=(6b67bef6-0aa1-11e8-b924-0a580a28023f, c339ff605be6113b9d344673d0d7b91a7c221766) already exists.
 [SQL: 'INSERT INTO testcase (id, job_id, hash, name, result, duration, message, repository_id) VALUES (%(id)s, %(job_id)s, %(hash)s, %(name)s, %(result)s, %(duration)s, %(message)s, %(repository_id)s)'] [parameters: ({'id': UUID('362ff5a2-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': '722f2a4733e630e4f23e8ded5d56811d1a0fafa8', 'name': 'Raven_Tests_ErrorHandlerBreadcrumbHandlerTest.testSimple', 'result': 1, 'duration': 4.369, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('363007c2-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': 'edd918d5190670d3a951145fec76e0a7106510b3', 'name': 'Raven_Tests_MonologBreadcrumbHandlerTest.testSimple', 'result': 1, 'duration': 3.694, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('363011d6-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': '58e2ae70e33d0d0e651ecf77bb2e661cda4ae991', 'name': 'Raven_Tests_MonologBreadcrumbHandlerTest.testErrorInMessage', 'result': 1, 'duration': 0.465, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('36301da2-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': 'b80c52e956f7f12e7a706f447551916ad7d1fa83', 'name': 'Raven_Tests_BreadcrumbsTest.testBuffer', 'result': 1, 'duration': 0.32, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('3630278e-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': 'c9bd830672d5e94b01b3e7e1657ef62c8deaa970', 'name': 'Raven_Tests_BreadcrumbsTest.testJson', 'result': 1, 'duration': 0.312, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('363030c6-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': '7a38b616209b3eadc7c52826cfa46b9c9a15a68b', 'name': 'testParseDSNWithDisabledValue with data set #0', 'result': 1, 'duration': 0.479, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('36303a26-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': '4f2af9f43d5a5c3ae8585cfa3afa566020fde4b7', 'name': 'testParseDSNWithDisabledValue with data set #1', 'result': 1, 'duration': 0.272, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('3630469c-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': '4c4452446b3a0b40023272b3c11d2ee92576dbd2', 'name': 'testParseDSNWithDisabledValue with data set #2', 'result': 1, 'duration': 0.266, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}  ... displaying 10 of 178 total bound parameter sets ...  {'id': UUID('36370aa4-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': 'e93d2f1b195d236c7ac4072544d035653fb2f986', 'name': 'Raven_Tests_UtilTest.testGetReturnsDefaultOnMissing', 'result': 1, 'duration': 0.319, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')}, {'id': UUID('363713c8-0aa4-11e8-9a65-0a580a281f12'), 'job_id': UUID('6b67bef6-0aa1-11e8-b924-0a580a28023f'), 'hash': 'cbbed71f0351e98a2d87bbfc6817915a8f758856', 'name': 'Raven_Tests_UtilTest.testGetReturnsPresentValuesEvenWhenEmpty', 'result': 1, 'duration': 0.25999999999999995, 'message': '', 'repository_id': UUID('ac28fdd4-ffba-11e7-b153-0a580a281308')})] (Background on this error at: http://sqlalche.me/e/gkpj)
```

**Comment by dcramer** [comment]
fwiw i dont think i've fixed the issue in zeus, so while it reports build results, it *doesnt* correctly report test results yet

**Comment by HazAT** [comment]
I am merging this now, for the new SDK we will use https://github.com/getsentry/craft anyway which uses Zues.

**Pull request merged**